gazelle/std_modules.py

```diff
@@ -3,30 +3,27 @@
 # it evaluates, it outputs true/false for whether the module is part of the
 # standard library or not.
 
-import site
+import os
 import sys
-
-
-# Don't return any paths, all userland site-packages should be ignored.
-def __override_getusersitepackages__():
-    return ""
-
-
-site.getusersitepackages = __override_getusersitepackages__
```

Comment:
This code wasn't having the desired effect.

Reply:
Interesting.